blog/tutorial/01-index.md

```diff
@@ -86,13 +85,13 @@ Posts in the key-value store will look like this:
 
 Right now the store is empty. Next, define how the user adds a posts.
 
-With the Cosmos SDK, users can interact with your app with either a CLI (`blogd`) or by sending HTTP requests. Let's define the CLI command first. Users should be able to type `blogd tx blog create-post 'This is a post!' 'Welcome to my blog app.' --from=user1` to add a post to your store. The `create-post` subcommand hasn’t been defined yet--let’s do it now.
+With the Cosmos SDK, users can interact with your app with either a CLI (`blogd`) or by sending HTTP requests. Let's define the CLI command first. Users should be able to type `blogd tx blog create-post 'This is a post!' 'Welcome to my blog app.' --from=alice` to add a post to your store. The `create-post` subcommand hasn’t been defined yet--let’s do it now.
```

Comment:
In default Starport v0.15+ creates `user1` and `user2` instead of `bob` and `alice` - which version of starport do you use?